### PR TITLE
Add new +php-laravel-mode snippets

### PR DESCRIPTION
**Pull request by elken**
Add a couple of snippets for default Laravel classes and for migrations

### CODE REVIEW

**Review by hlissner — APPROVED** [review]

File: +php-laravel-mode/.yas-parents.el [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+                buffer-file-name
+                (file-relative-name (doom-project-root))
+                file-name-directory
+                capitalize))
+        0
+        -1))
+
+(defun +php-laravel-mode--get-class-name ()
+    "Get a formatted class name for the current PHP file"
+    (string-join (mapcar 'capitalize (string-split-words (file-name-base buffer-file-name)))))
```
Comment (hlissner) [inline +php-laravel-mode/.yas-parents.el:26]: Please reindent (should be 2 spaces per level, not 4)

File: +php-laravel-mode/.yas-parents.el [hunk 2]
```diff
@@ -0,0 +1,26 @@
+(eval-when-compile (require 'subr-x))
+
+(defun string-split-words (string)
```
Comment (hlissner) [inline +php-laravel-mode/.yas-parents.el:3]: Please rename to `+php-laravel-mode--string-split-words` (or similar). Wordy, I know, but `string-split-words` stands too high a chance of conflicting with future APIs.

**Review by hlissner — CHANGES_REQUESTED** [review]

File: +php-laravel-mode/.yas-parents.el (hunk 2 above)
Comment (elken) [reply +php-laravel-mode/.yas-parents.el:3]: Yeah you're right, it's only used in 1 place so I might just inline it

File: +php-laravel-mode/.yas-parents.el (hunk 1 above)
Comment (elken) [reply +php-laravel-mode/.yas-parents.el:26]: That's how indent region is doing it now for some reason. Will try and edit manually

**Review by hlissner — APPROVED** [review]

**Comment by hlissner** [comment]
LGTM. Thanks for your help!

**Pull request merged**